README.md

```diff
@@ -81,41 +81,61 @@ By default we download all the libraries from GitHub and build them locally, so
 
 ### Configuring on Windows
 
+#### Installing the dependencies / environment
 Install [MS Visual Studio Community / Enterprise](https://visualstudio.microsoft.com/vs/community/), [Strawberry perl](https://strawberryperl.com/), and [Chocolatey](https://chocolatey.org/install) if not installed already
 
-Get the libraries by running the following in powershell
+Get the libraries by running the following in `powershell`
```

Comment:
```suggestion
Install dependency libraries by running the following in `powershell`
```

Reply:
Left as is, this line sounds likes it's talking about the above MSVC, Strawberry, Chocolatey "libraries", but it is not.

Reply:
It has the instructions right below. I don't think this can be misunderstood

Reply:
Requesting the change mostly for grammar/correctness sake. Online search yields that "the" should be used if "The noun is known to the speaker" or "The noun has been mentioned before".

Another approach could be to change it to:
"Get the following libraries by..."
This one is OK by me from a correctness standpoint but lacks the context that these are dependency libraries.

Reply:
Here, "the libraries" refers to the ones mentioned in the header